ownership/lookup.go

```diff
@@ -10,16 +10,16 @@ import (
 // It is intended to be used with the implementation on Os.  There's also one in mock.go that
 // uses fixed data instead of operating-system sourced data.
 type Lookup interface {
-	UID(username string) (uint32, error)
```

Comment:
Not sure why we're changing it from `uint32` to `int`. uid/gid are unsigned 32bit ints on POSIX systems, IIRC, but I don't know if it matters in reality.

Reply:
ultimately the point of these values are to call Chown, which takes int. 
 https://golang.org/pkg/os/#Chown